### ISSUE TITLE
LFO depth for square/pulse doesn't match ARIA

### ISSUE THREAD

**Issue by swesterfeld** [sfztools/sfizz#605]
Not sure if you consider this a bug, but I saw that sfizz LFOs for pulse/square are range [-1:1] whereas ARIA appears to be [-0.5:0.5]. Here is a test case
```
<region>
lfo1_freq=1
lfo1_wave=3
lfo1_pitch=600
sample=*sine
```

In ARIA this jumps 6 semitones (3 semitones up, 3 semitones down), whereas in sfizz this is a jump of one octave. This only affects square/pulse wave forms (2-5)

**Comment by jpcima** [comment]
If it's not matching to ARIA, it's definitely a bug.
I'll check it ASAP, thanks for reporting.

**Comment by jpcima** [comment]
> In ARIA this jumps 6 semitones (3 semitones up, 3 semitones down), whereas in sfizz this is a jump of one octave. This only affects square/pulse wave forms (2-5)

The current behavior of sfizz follows that of Cakewalk, but it turns out ARIA acts different.
Anyway we are following ARIA in priority, so I'm going to change it, and also review other shapes while at it.

**Comment by jpcima** [comment]
> whereas ARIA appears to be [-0.5:0.5].

In fact it's [0;1]. I have to modify the sine too, which has inverted phase.

**Comment by jpcima** [comment]
Fixed by #613
